Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/KlarnaTokenizationViewModel.swift

```diff
@@ -147,13 +147,13 @@ class KlarnaTokenizationViewModel: PaymentMethodTokenizationViewModel, ExternalP
                 throw err
             }
             
-            if settings.orderItems.isEmpty {
+            if (settings.orderItems ?? []).isEmpty {
                 let err = KlarnaException.missingOrderItems
                 _ = ErrorHandler.shared.handle(error: err)
                 throw err
             }
             
-            if !settings.orderItems.filter({ $0.unitAmount == nil }).isEmpty {
+            if !(settings.orderItems ?? []).filter({ $0.unitAmount == nil }).isEmpty {
```

Comment:
maybe the orderItems should default to an empty array so that we don't need to null check each time?

Reply:
I did that on purpose. This way we know that they haven't been set as long as they're nil, or that they have been set to an empty array (which means we have a response).